### PR TITLE
GM: remove redundant CarController class variable

### PR DESCRIPTION
**Pull request by sshane**
If we trust the CANParser, `CS.loopback_lka_steering_cmd_ts_nanos` should be 0 if it has never seen that message.

### CODE REVIEW

File: selfdrive/car/gm/carcontroller.py [hunk 1]
```diff
@@ -30,7 +30,6 @@ def __init__(self, dbc_name, CP, VM):
     self.cancel_counter = 0
 
     self.lka_steering_cmd_counter = 0
-    self.sent_lka_steering_cmd = False
```
Comment (adeebshihadeh) [inline selfdrive/car/gm/carcontroller.py:33]: this is not a good assumption. you can add valid/alive flag to the can parser if you want instead

Comment (adeebshihadeh) [reply selfdrive/car/gm/carcontroller.py:33]: you can also just check canValid

Comment (sshane) [reply selfdrive/car/gm/carcontroller.py:33]: the old assumption wasn't good or this one?

Comment (sshane) [reply selfdrive/car/gm/carcontroller.py:33]: made it more explicit by only updating CS's ts_nanos on explicit update of the message

**Pull request merged**